web/views.py

```
@@ -85,6 +84,13 @@
     Certificate,
     Challenge,
     ChallengeSubmission,
+    Quiz,
```

Ruff (0.8.2)

`.models.Quiz` imported but unused

Remove unused import

(F401)